main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import "fmt"
+
+type List struct {
+	productID   int
+	productName string
+	price       int
+}
+
+func getList() []List {
+	l1 := List{
+		productID:   1,
+		productName: "Latte",
+		price:       150,
+	}
+	l2 := List{
+		productID:   2,
+		productName: "Filter coffee",
+		price:       100,
+	}
+	return []List{l1, l2}
+}
+func main() {
+	fmt.Println("Welcome to Jo's coffee shop")
```

Comment:
We may need to reuse the name of the shop in a different part of the program. Please store the name of the store in a package-level constant.